capsules/classifier_face_age_gender_openvino/backend.py

```diff
@@ -11,6 +11,20 @@
 from . import config
 
 
+def _get_bin(number: float):
+    bins = config.age_bins
+    if len(bins) is 0:
```

Comment:
Don't use `is` with literals. Use `==`

https://wsvincent.com/python-wat-integer-cache/